bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
```

Comment:
In fact, the two templates in `/usr/share/bunsen/skel` will also need rewriting to cope with home directories other than `/home/username`.
Lines like `file:///home/%USERNAME%/Documents` will need to be `file://%USERHOME%/Documents` and that section of the script modified. (They are also in **bunsen-configs**, so it won't be difficult.)

BTW how reliable is the $HOME environment variable at this point? Can we not just accept that, along with $USER?